src/pages/tBTC/Bridge/DepositDetails.tsx

```diff
@@ -180,129 +179,111 @@ export const DepositDetails: PageComponent = () => {
-        isProcessCompleted={mintingProgressStep === "completed"}
-      >
-        {(isFetching || !data) && !error && (
-          <BridgeProcessDetailsPageSkeleton />
-        )}
-        {error && <>{error}</>}
-        {!isFetching && !!data && !error && (
-          <>
-            <Stack
-              direction={{
-                base: "column",
-                xl: "row",
-              }}
-              divider={<StackDivider />}
-              spacing={4}
-            >
-              <Flex flexDirection="column" w={{ base: "100%", xl: "65%" }}>
-                <Flex mb="4" alignItems="center" textStyle="bodyLg">
-                  <BodyLg>
-                    <Box
-                      as="span"
-                      fontWeight="600"
-                      color={depositStatusTextColor}
-                    >
-                      {mintingProgressStep === "completed"
-                        ? "Minted"
-                        : "Minting"}
-                    </Box>
-                    {mintingProgressStep !== "completed" && ` - In progress...`}
-                  </BodyLg>{" "}
-                  <InlineTokenBalance
-                    tokenAmount={amount || "0"}
-                    tokenSymbol="tBTC"
-                    withSymbol
-                    ml="auto"
-                  />
-                </Flex>
-                {mintingProgressStep !== "completed" && (
-                  <Alert status="info" my={6}>
-                    <AlertIcon />
-                    <AlertDescription>
-                      It is safe to close this window. Minting will continue as
-                      a background process and will not be interrupted.
-                    </AlertDescription>
-                  </Alert>
-                )}
-                <StepSwitcher />
+      {(isFetching || !data) && !error && <BridgeProcessDetailsPageSkeleton />}
+      {error && <>{error}</>}
+      {!isFetching && !!data && !error && (
+        <>
+          <Stack
+            direction={{
+              base: "column",
+              xl: "row",
+            }}
+            divider={<StackDivider />}
+            spacing={4}
+          >
+            <Flex flexDirection="column" w={{ base: "100%", xl: "65%" }}>
+              <Flex mb="4" alignItems="center" textStyle="bodyLg">
+                <BodyLg>
+                  <Box
+                    as="span"
+                    fontWeight="600"
+                    color={depositStatusTextColor}
+                  ></Box>
+                </BodyLg>{" "}
+                <InlineTokenBalance
+                  tokenAmount={amount || "0"}
+                  tokenSymbol="tBTC"
+                  withSymbol
+                  ml="auto"
+                />
               </Flex>
-              <Flex
-                w={{ base: "100%", xl: "35%" }}
-                mb={{ base: "20", xl: "unset" }}
-                direction="column"
+              {mintingProgressStep !== "completed" && (
```

Comment:
Let's use `DepositDetailsTimelineStep` type instead of writing "completed" as a plain string.

Reply:
`DepositDetailsTimelineStep` is a type definition and cannot be used here. Would you like to add corresponding enumeration structure?

Reply:
No, let's not block this PR for that. I'm resolving the conversation and we can merge.